test/test_srtp.c

```diff
@@ -146,9 +146,9 @@ void srtp_calc_aead_iv_srtcp_seq_num_over_0x7FFFFFFF_bad_param()
  */
 void srtp_calc_aead_iv_srtcp_distinct_iv_per_sequence_number()
 {
+#define SAMPLE_COUNT (3)
```

Comment:
Did clang-format move this or did you?

Reply:
I moved it. It messed with the comments, and having the def and undef as sort of a scope did not look too bad, so just left it like that. :-)

Reply:
Ah, OK.  I thought the tool was getting too smart.  Made me worry about what other "smart" decisions it might want to make.